StreamVideoTests/IntegrationTests/CallCRUDTests.swift

```diff
@@ -189,75 +226,186 @@ class CallCRUDTest: IntegrationTest {
-        let call = client.call(callType: "audio_room", callId: UUID().uuidString)
-        try await call.create(memberIds: ["thierry"])
+    func test_requestPermissionDiscard() async throws {
+        let firstUserCall = client.call(
+            callType: String.audioRoom,
+            callId: randomCallId
+        )
+        try await firstUserCall.create(memberIds: [user1])
 
-        let tommasoClient = getUserClient(id: "tommaso")
-        try await tommasoClient.connect()
-        let tommasoCall = tommasoClient.call(callType: "audio_room", callId: call.callId)
+        let secondUserClient = getUserClient(id: user2)
+        try await secondUserClient.connect()
+        let secondUserCall = secondUserClient.call(
+            callType: String.audioRoom,
+            callId: firstUserCall.callId
+        )
 
-        let _ = try await tommasoCall.get()
-        var hasAudioCapability = await tommasoCall.currentUserHasCapability(.sendAudio)
+        let _ = try await secondUserCall.get()
+        var hasAudioCapability = await secondUserCall.currentUserHasCapability(.sendAudio)
         XCTAssertFalse(hasAudioCapability)
-        var hasVideoCapability = await tommasoCall.currentUserHasCapability(.sendVideo)
+        var hasVideoCapability = await secondUserCall.currentUserHasCapability(.sendVideo)
         XCTAssertFalse(hasVideoCapability)
 
-        try await tommasoCall.request(permissions: [.sendAudio])
+        try await secondUserCall.request(permissions: [.sendAudio])
 
-        await assertNext(call.state.$permissionRequests) { value in
-            return value.count == 1 && value.first?.permission == "send-audio"
+        await assertNext(firstUserCall.state.$permissionRequests) { value in
+            return value.count == 1 && value.first?.permission == Permission.sendAudio.rawValue
         }
-        if let p = await call.state.permissionRequests.first {
+        if let p = await firstUserCall.state.permissionRequests.first {
             p.reject()
         }
         
         // Test: permission requests list is now empty
-        await assertNext(call.state.$permissionRequests) { value in
+        await assertNext(firstUserCall.state.$permissionRequests) { value in
             return value.count == 0
         }
 
-        hasAudioCapability = await tommasoCall.currentUserHasCapability(.sendAudio)
+        hasAudioCapability = await secondUserCall.currentUserHasCapability(.sendAudio)
         XCTAssertFalse(hasAudioCapability)
-        hasVideoCapability = await tommasoCall.currentUserHasCapability(.sendVideo)
+        hasVideoCapability = await secondUserCall.currentUserHasCapability(.sendVideo)
         XCTAssertFalse(hasVideoCapability)
     }
     
-    func test_mute_user_by_id() async throws {
-        let call = client.call(callType: "audio_room", callId: UUID().uuidString)
-        try await call.create(memberIds: ["thierry", "tommaso"])
+    // TODO: add assertion
+    func test_muteUserById() async throws {
```

Comment:
similarly, call.get and check in the session if that user is muted.

Reply:
Done 👍